include/zephyr/bluetooth/conn.h

```diff
@@ -479,6 +479,50 @@ struct bt_conn_le_tx_power {
 	int8_t max_level;
 };
 
+enum bt_conn_le_tx_power_report_reason {
+	/** Local Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_LOCAL_CHANGED = 0,
+
+	/** Remote Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_REMOTE_CHANGED = 1,
+
+	/** HCI_LE_Read_Remote_Transmit_Power_Level command completed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_READ_REMOTE_COMPLETED = 2,
+};
```

Comment:
I feel it is beneficial to have a description of the values defined like this, so I would just add a comment referencing the HCI specification? Thinking about your other comment this would be a bit contradictory to reference though(as conn.h is not tied to hci).

Reply:
> I feel it is beneficial to have a description of the values defined like this, so I would just add a comment referencing the HCI specification? Thinking about your other comment this would be a bit contradictory to reference though(as conn.h is not tied to hci).

I agree, it's beneficial. There should at least be a comment for the `reason` struct member that it contains HCI values. As for the values themselves, I'm thinking maybe `hci.h` is a better place for them (near the command response parameter definition) rather than `conn.h`?

Reply:
I added a core spec reference, imo core reference is the cleanest way to show where it comes from and the user can easily determine if it is outdated or not. If you though about something else, let me know :)